evaluate/report/collection.go

```diff
@@ -20,6 +20,9 @@ type AssessmentPerModelPerLanguagePerRepository map[model.Model]map[language.Lan
 // AssessmentPerModel holds a collection of assessments per model.
 type AssessmentPerModel map[model.Model]metrics.Assessments
 
+// AssessmentPerLanguagePerModel holds a collection of assessments per language and model.
+type AssessmentPerLanguagePerModel map[language.Language]map[model.Model]metrics.Assessments
```

Comment:
I would've changed the order to keep the model up front. It shouldn't really matter, right? Or is this something that Markus' hacky version explicitly did?

Reply:
I guess it will only change the ordering in the CSV file in the end. And sorting by language first might be nice.